server.js

```diff
@@ -323,9 +329,21 @@ export async function startServer() {
   }
 
   const httpServer = app.listen(PORT, () => {
-    console.log(
-      `Server started at ${http2 ? "https" : "http"}://localhost:${PORT}`,
-    );
+    const scheme = http2 ? "https" : "http";
+    const url = `${scheme}://localhost:${PORT}`;
+    console.log(`Server started at ${url}`);
+    // Auto open browser
+    if (devMode && OPEN_BROWSER_ON_START) {
```

Comment:
I wouldn't gate auto open behind the server being in devmode so authors in content are able to enable it:
```suggestion
    if (OPEN_BROWSER_ON_START) {
```